tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
i'll assume it's implicit that the comments on the other, similar ts pr you've been working on will get applied here